src/test/java/io/github/ryotan/code/example/UserType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * These codes are licensed under CC0.
+ *
+ * http://creativecommons.org/publicdomain/zero/1.0/deed
+ * http://creativecommons.org/publicdomain/zero/1.0/deed.ja
+ */
+
+package io.github.ryotan.code.example;
+
+import io.github.ryotan.code.CodeEnum;
+
+public enum UserType implements CodeEnum<UserType> {
+    ADMINISTRATOR("1", "管理者"),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String "?????????" appears 3 times in the file.](https://www.codacy.com/app/ryotan/code-enum-support/file/3935988752/issues/source?bid=3375378&fileBranchId=3375797#l13)